docs/src/rules/no-constant-condition.md

```diff
@@ -134,19 +214,34 @@ while (true) {
     }
 };
 
-for (;true;) {
+do {
+    doSomething();
+    if (condition()) {
+        break;
+    }
+} while (true)
+```
+
+:::
+
+::: correct
+
+```js
+/*eslint no-constant-condition: ["error", { "checkLoops": "none" }]*/
```

Comment:
Did you mean `"checkLoops": false`? Because `"checkLoops": "none"` is used in the previous block.

```suggestion
/*eslint no-constant-condition: ["error", { "checkLoops": false }]*/
```